src/main/java/ch/njol/skript/effects/EffSendTitle.java

```diff
@@ -32,90 +34,80 @@
 	"send subtitle \"Party!\" to all players"
 })
-@Since("2.3")
+@Since("2.3, INSERT VERSION (object support)")
 public class EffSendTitle extends Effect {
-	
-	private final static boolean TIME_SUPPORTED = Skript.methodExists(Player.class,"sendTitle", String.class, String.class, int.class, int.class, int.class);
-	
 	static {
-		if (TIME_SUPPORTED)
-			Skript.registerEffect(EffSendTitle.class,
-					"send title %string% [with subtitle %-string%] [to %players%] [for %-timespan%] [with fade[(-| )]in %-timespan%] [[and] [with] fade[(-| )]out %-timespan%]",
-					"send subtitle %string% [to %players%] [for %-timespan%] [with fade[(-| )]in %-timespan%] [[and] [with] fade[(-| )]out %-timespan%]");
-		else
-			Skript.registerEffect(EffSendTitle.class,
-					"send title %string% [with subtitle %-string%] [to %players%]",
-					"send subtitle %string% [to %players%]");
+		Skript.registerEffect(EffSendTitle.class,
+			"send title %object% [with subtitle %-object%] [to %players%] [for %-timespan%] [with fade[(-| )]in %-timespan%] [[and] [with] fade[(-| )]out %-timespan%]",
+			"send subtitle %object% [to %players%] [for %-timespan%] [with fade[(-| )]in %-timespan%] [[and] [with] fade[(-| )]out %-timespan%]");
 	}
-	
-	@Nullable
-	private Expression<String> title;
-	@Nullable
-	private Expression<String> subtitle;
-	@SuppressWarnings("null")
-	private Expression<Player> recipients;
-	@Nullable
-	private Expression<Timespan> fadeIn, stay, fadeOut;
-	
-	@SuppressWarnings({"unchecked", "null"})
+
+	private @Nullable Expression<?> title;
+	private @Nullable Expression<?> subtitle;
+	private @Nullable Expression<Player> recipients;
+	private @Nullable Expression<Timespan> fadeIn, stay, fadeOut;
+
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
-		title = matchedPattern == 0 ? (Expression<String>) exprs[0] : null;
-		subtitle = (Expression<String>) exprs[1 - matchedPattern];
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parser) {
+		if (matchedPattern == 0) title = LiteralUtils.defendExpression(exprs[0]);
+		subtitle = LiteralUtils.defendExpression(exprs[1 - matchedPattern]);
 		recipients = (Expression<Player>) exprs[2 - matchedPattern];
-		if (TIME_SUPPORTED) {
-			stay = (Expression<Timespan>) exprs[3 - matchedPattern];
-			fadeIn = (Expression<Timespan>) exprs[4 - matchedPattern];
-			fadeOut = (Expression<Timespan>) exprs[5 - matchedPattern];
+		stay = (Expression<Timespan>) exprs[3 - matchedPattern];
+		fadeIn = (Expression<Timespan>) exprs[4 - matchedPattern];
+		fadeOut = (Expression<Timespan>) exprs[5 - matchedPattern];
+		if (title != null && subtitle != null) {
+			return LiteralUtils.canInitSafely(title, subtitle);
+		} else if (title != null) {
+			return LiteralUtils.canInitSafely(title);
+		} else {
+			return LiteralUtils.canInitSafely(subtitle);
 		}
```

Comment:
can simplify a little with
```suggestion
		if (title != null && !LiteralUtils.canInitSafely(title))
			return false;
		if (subtitle != null && !LiteralUtils.canInitSafely(subtitle))
			return false;
		return true;
```